tests/functional/object/mcg/ui/test_bucket_policy_ui.py

```diff
@@ -0,0 +1,232 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.pytest_customization.marks import (
+    black_squad,
+    tier1,
+    post_upgrade,
+    polarion_id,
+)
+from ocs_ci.ocs.ui.page_objects.buckets_tab import BucketsTab
+from ocs_ci.ocs.ui.page_objects.bucket_tab_permissions import (
+    BucketsTabPermissions,
+    PolicyConfig,
+    PolicyType,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
```

Comment:
please move the polarion decorator to under each permutation of `pytest.mark.parametrize`.
I suggest keeping only 1 permutation in tier1 and changing the rest to tier2 or tier3